src/Compilers/Core/Portable/WellKnownMember.cs

```diff
@@ -482,6 +482,10 @@ internal enum WellKnownMember
         System_Runtime_CompilerServices_ITuple__get_Item,
         System_Runtime_CompilerServices_ITuple__get_Length,
 
+        System_InvalidOperationException__ctor,
+        System_MatchFailureException__ctor0,
```

Comment:
>System_MatchFailureException__ctor0 [](start = 8, length = 35)

To follow established pattern, consider using name "System_MatchFailureException__ctor".